nixos/modules/tasks/auto-upgrade.nix

```diff
@@ -91,24 +112,36 @@ let cfg = config.system.autoUpgrade; in
       path = with pkgs; [ coreutils gnutar xz.bin gzip gitMinimal config.nix.package.out ];
 
       script = let
-          nixos-rebuild = "${config.system.build.nixos-rebuild}/bin/nixos-rebuild";
-        in
+        nixos-rebuild = "${config.system.build.nixos-rebuild}/bin/nixos-rebuild";
+        date     = "${pkgs.coreutils}/bin/date";
```

Comment:
Shouldn't `date`, `readlink` and `shutdown` already be available, since `coreutils` is in the unit's `path`?

Reply:
I guess. I figured it wouldn't hurt to make the paths explicit, but I'm fine removing this if that's they way things are usually done.